cosmoz-image-viewer.html

```diff
@@ -136,11 +136,11 @@
 
 		</style>
 
-		<div class="nav counter" hidden$="[[!_showPageNumber]]">
+		<div class="nav counter hide-on-small-height" hidden$="[[!_showPageNumber]]">
```

Comment:
This breaks behavior of `show-page-number` attribute, as when the height is > 100, hidden attribute will be removed.